lib/client/client.js

```diff
@@ -131,11 +131,11 @@ Client.prototype.requestAll = function (method, uri) {
     bodyList.push(body);
     responseList.push(response);
     resultList.push(result);
-    nextPage = result.next_page;
+    nextPage = typeof result.next_page === 'undefined' ? null : result.next_page;
```

Comment:
i think the error in #83 is due to ```result``` being undefined, causing the ```result.next_page``` call in the condition to crash the program.
Maybe refactor to:
```nextPage =  result ? result.next_page : null;``` ?